services/121-service/src/registration-events/registration-events.service.spec.ts

```diff
@@ -131,50 +89,6 @@ describe('RegistrationEventsService', () => {
     expect(registrationEventsService).toBeDefined();
   });
 
-  it('should return events in json dto format', async () => {
-    // Act
-    const result = await registrationEventsService.getEventsAsJson({
-      programId,
-      searchOptions: {},
-    });
-
-    const resultEvent = result[0];
-    expect(resultEvent.id).toBe(mockFindEventResult[0].id);
-    expect(resultEvent.created).toBe(mockFindEventResult[0].created);
-    expect(resultEvent.user?.id).toBe(mockFindEventResult[0].user.id);
-    expect(resultEvent.user?.username).toBe(
-      mockFindEventResult[0].user.username,
-    );
-    expect(resultEvent.registrationId).toBe(
-      mockFindEventResult[0].registration.id,
-    );
-
-    const expectedAttributes = {
-      [attributeEntityFieldName.key]: attributeEntityFieldName.value,
-      [attributeEntityOldValue.key]: attributeEntityOldValue.value,
-      [attributeEntityNewValue.key]: attributeEntityNewValue.value,
-    };
-    expect(resultEvent.attributes).toEqual(expectedAttributes);
-  });
-
-  it('should return events in flat dto format (which is used for excel export)', async () => {
-    // Act
-    const result = await registrationEventsService.getEventsAsXlsx({
-      programId,
-      searchOptions: {},
-    });
-
-    const resultEvent = result[0];
-    expect(resultEvent.changedAt).toBe(mockFindEventResult[0].created);
-    expect(resultEvent.changedBy).toBe(mockFindEventResult[0].user.username);
-    expect(resultEvent.fieldName).toBe(attributeEntityFieldName.value);
-    expect(resultEvent.oldValue).toBe(attributeEntityOldValue.value);
-    expect(resultEvent.newValue).toBe(attributeEntityNewValue.value);
-    expect(resultEvent.paId).toBe(
-      mockFindEventResult[0].registration.registrationProgramId,
-    );
-  });
```

Comment:
now that the service method no longer distinguishes between xlsx and json (flat format is returned either way), I thought having tests on these different paths was no longer valuable / worth the effort to fix.